frontend/src/pages/LandlordPage.tsx

```diff
@@ -1,12 +1,44 @@
 import React, { ReactElement } from 'react';
 import { useParams } from 'react-router-dom';
+import Review from '../components/Review';
+
+let reviews = [
+  {
+    name: 'user1',
+    overall_rating: 3,
+    date: new Date(),
+    text:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam auctor mauris a scelerisque rhoncus. Nam vitae lacus at neque faucibus porttitor. Phasellus mollis maximus neque, vehicula consectetur enim sagittis ac. Sed viverra risus nibh, non pulvinar mauris fermentum sed. Praesent pellentesque dapibus felis nec interdum. ',
+  },
+  {
+    name: 'user2',
+    overall_rating: 2,
+    date: new Date(),
+    text:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam auctor mauris a scelerisque rhoncus. Nam vitae lacus at neque faucibus porttitor. Phasellus mollis maximus neque, vehicula consectetur enim sagittis ac. Sed viverra risus nibh, non pulvinar mauris fermentum sed. Praesent pellentesque dapibus felis nec interdum. ',
+  },
+  {
+    name: 'user3',
+    overall_rating: 1,
+    date: new Date(),
+    text:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam auctor mauris a scelerisque rhoncus. Nam vitae lacus at neque faucibus porttitor. Phasellus mollis maximus neque, vehicula consectetur enim sagittis ac. Sed viverra risus nibh, non pulvinar mauris fermentum sed. Praesent pellentesque dapibus felis nec interdum. ',
+  },
+];
 
 const LandlordPage = (): ReactElement => {
   const { landlordId } = useParams<Record<string, string | undefined>>();
 
   return (
     <div>
       <h1>{`This is dummy text! My current landlordId is ${landlordId}`}</h1>
+      {reviews.map((review) => (
```

Comment:
Here, you can destructure `review` to be `{ date, text }`. This way, you don't have to do `review.date` and `review.text` and instead just use them as `date` and `text` respectively.